billing.example.yaml

```diff
@@ -33,6 +33,10 @@ billables:
     unit_price:  10.23
     currency:    €
 
+# You may enter tax as a percentage here (from 0 to 1.0).
+tax:
+  percentage: 0.1
```

Comment:
We really need to support a default tax and an overridable tax rate per item. Some items may be taxable at different rates.